### PR TITLE
[SDPT-694] Change Helm Chart to one application per cluster for workshop-sample

### PR DESCRIPTION
**Pull request by youminhan**
1. Change the helm chart structure to use one Flink cluster per Flink app.
2. Test the application on SDP.
3. Update the README with the latest instruction.

Signed-off-by: Youmin Han <Youmin.Han@dell.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,11 +117,23 @@ kubectl port-forward service/repo 9092:80 --namespace workshop-samples
...
 ```
--   Build and publish your application JAR file.
+-   Build and publish your application JAR file. Make sure to replace the `release name` and `application values file` with appropriate values.
 ```
 ./gradlew publish
-helm upgrade --install --timeout 600s jsonreader \
---wait --namespace workshop-samples charts
+helm upgrade --install --timeout 600s --wait \
+    <Release Name> \
+    charts/workshop-samples \
```
Comment (claudiofahey) [inline README.md:125]: Is `workshop-samples` a good name for this chart? This chart is general-purpose but only for Flink jobs. Spark and other types of apps will need to use different charts. So you might want to rename the chart to `flink-app`, `workshop-samples-flink-app`, or `sdp-flink-app`.

File: values/flink-image-reader.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  job.disableCheckpoint: "false"
+  job.disableOperatorChaining: "true"
+  rebalance: "false"
+
+mainClass: com.dellemc.oe.readers.ImageReader
+
+mavenCoordinate:
+  artifact: stream-readers
+  group: com.dellemc.oe.readers
+  version: 0.1.0
```
Comment (claudiofahey) [inline values/flink-image-reader.yaml]: Include LF at end of all files.

**Review by claudiofahey — APPROVED** [review]
Looks good but you should rename the chart as explained in the comment.

File: README.md (hunk 1 above)
Comment (youminhan) [reply README.md:125]: Update the chart using `flink-app`

File: values/flink-image-reader.yaml (hunk 2 above)
Comment (youminhan) [reply values/flink-image-reader.yaml]: Add `\n` at the end of each `yaml` file

**Pull request merged**